packages/viewer/src/components/viewer-pin-group/viewer-pin-group.tsx

```diff
@@ -141,7 +141,7 @@ export class ViewerPinGroup {
           data-testid={`drawn-pin-${this.pin.id}`}
           position={this.pin.worldPosition}
           onPointerDown={(e) => {
-            if (e.buttons !== 2) {
+            if (this.pinController?.getToolMode() === 'edit') {
```

Comment:
Ideally we won't have to stop propagation, but in edit mode, the text label has not submitted, so dragging an anchor causes the latest text to not be saved.